docs/enterprise-guide/dashboard-subscriptions.md

```diff
@@ -0,0 +1,28 @@
+
+## Customize filter values for each dashboard subscription
+
+This page covers the Enterprise Edition features for dashboard subscriptions. Our User's Guide can give you an introduction to [dashboard subscriptions](../users-guide/dashboard-subscriptions.md). 
+
+Enterprise Edition lets you set up multiple subscriptions for a single dashboard, and lets you customize which filter values to apply to each subscription. That way you can send one group of people an email (or Slack message) the contents of the dashboard with one filter value applied, and set up another subscription to send the dashboard's contents with a different filter applied. You only need to maintain one dashboard, and you can set up different subscriptions to give different groups of people the info they need. 
```

Comment:
> lets you set up multiple subscriptions for a single dashboard

Let's be careful not to give folks the mistaken impression that you can only create multiple subscriptions in EE. That's not the case.

> and set up another subscription to send the dashboard's contents with a different filter applied

Feels a bit repetitive, syntactically.

> Enterprise Edition

IDK what's right, but I've in the past tended to write and say "_the_ Enterprise Edition."